src/components/triniti-app-nav/MainNavContent.jsx

```diff
@@ -31,6 +54,26 @@ class MainNavContent extends React.Component {
     }
   }
 
+  handleToggleModal(id) {
+    this.setState({
+      activeModalId: this.state.activeModalId ? '' : id,
+    });
+  }
+
+  renderLinkModalPopup(dpLink) {
+    const { history } = this.props;
+    const { linkTitle, modal: ModalComponent, modalId } = dpLink;
+    const id = modalId || kebabCase(linkTitle);
+    const isModalOpen = this.state.activeModalId === id;
+
+    return ([
+      <a key="button" href="#" className="dropdown-item" onClick={() => this.handleToggleModal(id)}>
```

Comment:
can just remove the href attr and it won't append the hash to the current window url